features/GettingStarted.md

```diff
@@ -1,19 +1,19 @@
-Install Rails-3
+Install Rails 6
 
-    $ gem install rails -v "~> 3.0.0"
+    $ gem install rails -v "~> 6.0.0"
 
 ### Generate an app
 
-    $ rails new example
-    $ cd example
+    $ rails new example_app
+    $ cd example_app
 
 ### Add rspec-rails to the Gemfile
 
-    $ echo 'gem "rspec-rails", :group => [:development, :test]' >> Gemfile
+    $ echo 'gem "rspec-rails", group: [:development, :test]' >> Gemfile
 
 ### Install the bundle
 
-    $ bundle install
```

Comment:
`install` is not enough as it doesn't add to `Gemfile.lock`.

Reply:
Can you clarify? I thought `bundle` was short for `bundle install` and should be sufficient for a new app...

Reply:
> Install the gems specified in your Gemfile(5). If this is the first time you run bundle install (and a Gemfile.lock does not exist), Bundler will fetch all remote sources, resolve dependencies and install all needed gems.

You're right. I had an impression that `bundle install` is not supposed to create/update `Gemfile.lock`. It seems that it's only the case for `bundle install --frozen`.
Will revert this.